microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/commons/MicroProfileJavaCodeActionParams.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.

```

Comment:
Do we need the 2020 header and "SPDX-License-Identifier: EPL-2.0" in this file?

Reply:
SPDX-License-Identifier: EPL-2.0 should be in all files

Year should be up-to-date in new files

Reply:
fixed, I change that for all classes coming from `com.redhat.microprofile.commons` package. Please create an issue for fixing this header for other files.